src/fix-install-headers.sh

```diff
@@ -2,4 +2,4 @@
 
 dir=$1
 find "$dir/include/cvc4/" -type f | \
-  xargs sed -i 's/include.*"\(.*\)"/include <cvc4\/\1>/'
+  xargs sed -i'' 's/include.*"\(.*\)"/include <cvc4\/\1>/'
```

Comment:
To make it work on the current version of macOS, I needed to do:

```suggestion
  xargs sed -i'' -e 's/include.*"\(.*\)"/include <cvc4\/\1>/'
```

This should work on 10.9+ [according to StackOverflow](https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux).

Side note: instead of piping to `xargs`, we should just use the `-exec` option of `find` here.

Reply:
Done.